source/help/getting-started/setting-your-status-availability.rst

```diff
@@ -0,0 +1,58 @@
+Setting Your Status and Availability
+====================================
+
+Let your team know whether you're available by setting a custom status and your availability in Mattermost.
+
+Setting a Custom Status
+-----------------------
+
+Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the left-hand sidebar, within conversations, and within Direct Messages. 
```

Comment:
left-hand > channel

Reply:
Fixed. Docs ticket https://mattermost.atlassian.net/browse/MM-33689 logged to review and update the full doc set.